### PR TITLE
[Issue #40] Patient profile

### PR DESCRIPTION
**Pull request by AlicjaKowalska**
Added PatientProfileActivity.kt as well as an funcionality of adding patient information to the database using AddPatientDataFragment.kt.

### CODE REVIEW

**Review by kozych — APPROVED** [review]

File: app/src/main/java/com/example/patientassistant/Fragment/AddPatientDataFragment.kt [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+        val formattedTimeDate = timeFormatDate.format(currentDate.time)
+        binding.date.setText(formattedTimeDate.toString())
+
+        val currentHour = Calendar.getInstance()
+        val timeFormatHour = SimpleDateFormat("HH:mm")
+        val formattedTimeHour = timeFormatHour.format(currentHour.time)
+        binding.hour.setText(formattedTimeHour.toString())
+    }
+
+    fun setAboutYouView() {
```
Comment (kozych) [inline app/src/main/java/com/example/patientassistant/Fragment/AddPatientDataFragment.kt:79]: I think we can have multiple fragments for this. That way will be easier to implement navigation component

File: app/src/main/java/com/example/patientassistant/Fragment/AddPatientDataFragment.kt [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+            val simpleDateFormat = SimpleDateFormat(formatPresentDate, Locale.getDefault())
+            binding.birthDate.setText(simpleDateFormat.format(calendar.time).toString())
+        }
+
+        val genderArray = arrayOf("Male", "Female", "Other")
+        val adapter = ArrayAdapter(requireActivity(), R.layout.simple_spinner_item, genderArray)
+        adapter.setDropDownViewResource(R.layout.simple_spinner_dropdown_item)
+        binding.genderSpinner.adapter = adapter
+
+        binding.addButton.setOnClickListener {
```
Comment (kozych) [inline app/src/main/java/com/example/patientassistant/Fragment/AddPatientDataFragment.kt:114]: Please also close fragment here

File: app/src/main/java/com/example/patientassistant/Fragment/AddPatientDataFragment.kt (hunk 1 above)
Comment (AlicjaKowalska) [reply app/src/main/java/com/example/patientassistant/Fragment/AddPatientDataFragment.kt]: Done

File: app/src/main/java/com/example/patientassistant/Fragment/AddPatientDataFragment.kt (hunk 2 above)
Comment (AlicjaKowalska) [reply app/src/main/java/com/example/patientassistant/Fragment/AddPatientDataFragment.kt]: Done

File: app/src/main/res/layout/activity_patient_profile.xml [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+        app:itemTextColor="@drawable/selector"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:menu="@menu/menu_navigation" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:background="#FAFFFFFF"
```
Comment (kozych) [inline app/src/main/res/layout/activity_patient_profile.xml:27]: Please use themes for background. Turning on night mode will not change this value

Comment (AlicjaKowalska) [reply app/src/main/res/layout/activity_patient_profile.xml]: Done

**Review by piotrgos45 — APPROVED** [review]

**Review by kozych — APPROVED** [review]

File: .idea/misc.xml [hunk 4]
```diff
@@ -7,4 +7,11 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
+  <component name="VisualizationToolProject">
```
Comment (michalmobi) [inline .idea/misc.xml:10]: this file should be ignored by .gitignore file. Please try to ignore this change.

**Review by michalmobi — APPROVED** [review]
There is a lot of changes in this PR. Please add some more description to the ticket  #40. That description: "create fragment / activity for patient profile" suggests that only one fragment will be added in the PR, but there are many. Please update it.

File: .idea/misc.xml (hunk 4 above)
Comment (AlicjaKowalska) [reply .idea/misc.xml]: done

**Pull request merged**